tests/draft-next/optional/refOfUnknownKeyword.json

```diff
@@ -2,7 +2,7 @@
     {
         "description": "reference of a root arbitrary keyword ",
         "schema": {
-            "$schema": "https://json-schema.org/draft/next/schema",
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
This, and other $schema declarations in this file, should be `draft-next`.

Reply:
Resolved